jsk_perception/CMakeLists.txt

```diff
@@ -228,6 +229,7 @@ jsk_add_nodelet(src/project_image_point.cpp "jsk_perception/ProjectImagePoint" "
 jsk_add_nodelet(src/sliding_window_object_detector.cpp "jsk_perception/SlidingWindowObjectDetector" "sliding_window_object_detector")
 jsk_add_nodelet(src/gaussian_blur.cpp "jsk_perception/GaussianBlur" "gaussian_blur")
 jsk_add_nodelet(src/kmeans.cpp "jsk_perception/KMeans" "kmeans")
+jsk_add_nodelet(src/darknet_object_detection "jsk_perception/DarknetObjectDetection" "darknet_object_detection")
```

Comment:
do we need `+add_darknet_definitions(darknet_object_detection)` for standalone node?